app/views/works/_work_form_associations_language.html.erb

```diff
@@ -1,6 +1,6 @@
 <% # expects local variable f %>
 <dt class="language required">
-  <%= f.label :language_id, ts("Choose a language") + "*" %>
+  <%= f.label :language_id, t("works.associations.language.choose") + "*" %>
```

Comment:
```suggestion
  <%= f.label :language_id, "#{t("works.associations.language.choose")} *" %>
```

Reply:
Actually, we do want the asterisk separated from the translatable text here! The required field indicators shouldn't be translatable.

Reply:
Oh, wait, sorry, I missed the closing parentheses. Carry on and ignore me! 🤦